content/Reducing_Power_Consumption.md

```diff
@@ -19,10 +19,10 @@ more.
 When waiting for a vblank event, this would be a BAD example:
 
 ```
- @@wait:
-  ld   a,(0FF44h)      ;LY
+ .wait:
+  ld   a,[0FF44h]      ;LY
```

Comment:
This should be `$FF44`; also, the preferred style is 1 space after commas and semicolons.

Reply:
👍